helix-core/src/main/java/org/apache/helix/messaging/handling/HelixTaskExecutor.java

```diff
@@ -488,11 +490,11 @@ public boolean scheduleTask(MessageTask task) {
               "Message handling task already sheduled for " + taskId, manager);
         }
       }
-    } catch (Exception e) {
+    } catch (Throwable e) {
       LOG.error("Error while executing task. " + message, e);
       _statusUpdateUtil
-          .logError(message, HelixTaskExecutor.class, e, "Error while executing task " + e,
-              manager);
+              .logError(message, HelixTaskExecutor.class, e, "Error while executing task. " + e,
```

Comment:
wondering if we can use the existing .logError as you are already printing 'e' as part of the additionalInfo.

Reply:
fixed this as part of other comment